### ISSUE TITLE
[dagster-celery-k8s] Pod names do not match engine event logs

### ISSUE THREAD

**Issue by catherinewu** [dagster-io/dagster#2713]
In the Engine Event logs, the pod names are of the form dagster-run-{run_id} but when inspecting kubernetes, the pod names are of the form dagster-run-{run_id}-{some random(?) 5 digit string}

**Comment by catherinewu** [comment]
chatted with nate, decided we should remove the pod names from the engine events since they are currently inaccurate (and we don't know the pod name until the job creates the pods)

**Comment by johannkm** [comment]
@catherinewu @alangenfeld on Monday I argued against removing pod names from the log because I thought it would have helped debug the issue with pod restarts that Sashank was working on. I was wrong, the log I was thinking about would have to come from within the step job, if it included the pod where it's running with it's logs. That would require some k8s specific logic in the `execute_run` call. From the executor, we could log the response from `get_pod_names_in_job`, but we couldn't attribute individual step events to a certain pod. 

Long story short I think removing seems good, if the user has the job name then they can query k8s and get the same amount of info. @catherinewu did you have a diff out for removing it already? I can make it if not
